src/API/Query.php

```diff
@@ -66,6 +66,8 @@ class Query
 //    /** @var bool */
 //    private $returnBinary = false;
 
+    private $restrictNavigation;
```

Comment:
Should specify the type using /** @var RestrictNavigation */ and import. As opposed to Java, all type expectation in PHP is done through metadata stored by comments.